components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/RefreshGrantHandlerTestOld.java

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall fix the license headers?

Reply:
https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2581/commits/6e5f72362aec0df8e2f727f80941b7a25c47b17f